### PR TITLE
task/0021-CW Multiplication table for number

### PR DESCRIPTION
**Pull request by Rascass**

### CODE REVIEW

**Comment by WhatWasThatt** [comment]
why did you add 3 files in one commit?

File: jest/multiTable/multiTable.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  it('should not contains \\n in the end', () => {
+    expect(multiTable(4).endsWith('\n')).toBeFalsy();
+  });
+  it('should return a table for 5', () => {
+    expect(multiTable(5)).toEqual('1 * 5 = 5\n2 * 5 = 10\n3 * 5 = 15\n4 * 5 = 20\n5 * 5 = 25\n6 * 5 = 30\n7 * 5 = 35\n8 * 5 = 40\n9 * 5 = 45\n10 * 5 = 50');
+  });
+  it('should return a table for 6', () => {
+    expect(multiTable(6)).toEqual('1 * 6 = 6\n2 * 6 = 12\n3 * 6 = 18\n4 * 6 = 24\n5 * 6 = 30\n6 * 6 = 36\n7 * 6 = 42\n8 * 6 = 48\n9 * 6 = 54\n10 * 6 = 60');
+  });
+});
```
Comment (WhatWasThatt) [inline jest/multiTable/multiTable.test.js:16]: why don't you have two negative tests?

**Comment by Rascass** [comment]
> why did you add 3 files in one commit?
fixed

**Review by WhatWasThatt — APPROVED** [review]

**Comment by WhatWasThatt** [comment]
thanks for your changes. keep up the good work!)

File: tasks/multiTable/multiTable.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports.multiTable = (number) => {
+  const sequence = Array.from(Array(11).keys());
+  const tableReducer = (accum, curValue) => String(accum).concat(`${curValue} * ${number} = ${curValue * number}\n`);
+  const multiTable = sequence.reduce(tableReducer);
```
Comment (mTeeeur) [inline tasks/multiTable/multiTable.js:4]: We can split it into one line

**Review by mTeeeur — CHANGES_REQUESTED** [review]

**Review by WhatWasThatt — APPROVED** [review]

File: tasks/multiTable/multiTable.js (hunk 2 above)
Comment (Rascass) [reply tasks/multiTable/multiTable.js]: fixed

**Review by mTeeeur — APPROVED** [review]
looks good

**Review by WhatWasThatt — APPROVED** [review]
good job

**Review by askomar — APPROVED** [review]
Ok. But you must name all your commits in correct way. And your file naming must be similar with README

**Pull request merged**